external/multi-base-images/flink/Dockerfile

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM openjdk:8-jre
```

Comment:
@RayRoestenburg @olofwalker missed this one. Spark uses `adoptopenjdk/openjdk8` as Lagom does. We should alignour java base images here.

Reply:
@skonto I think we need to talk about this before making a move, would be nice to know what the pro/cons are